client/app/core/rbac.service.js

```diff
@@ -21,4 +32,91 @@ export function RBACFactory() {
   function all() {
     return features;
   }
+
+  function setNavigation(productFeatures) {
+    const features = {
+      dashboard: {show: entitledForDashboard(productFeatures)},
+      services: {show: entitledForServices(productFeatures)},
+      orders: {show: entitledForServices(productFeatures)},
+      requests: {show: entitledForRequests(productFeatures)},
+      catalogs: {show: entitledForServiceCatalogs(productFeatures)},
+      dialogs: {show: entitledForService(productFeatures)},
+      administration: {show: entitledForService(productFeatures)},
+    };
+
+    return features;
+  }
+  function setNavFeatures(features) {
+    navFeatures = features;
+  }
+  function setActionFeatures(features) {
+    actionFeatures = features;
+  }
+  function getActionFeatures() {
+    return actionFeatures;
+  }
+  function setActions(productFeatures) {
+    const features = {
+      serviceView: {show: angular.isDefined(productFeatures.service_view)},
+      serviceEdit: {show: angular.isDefined(productFeatures.service_edit)},
+      serviceTag: {show: angular.isDefined(productFeatures.service_tag)},
+      serviceDelete: {show: angular.isDefined(productFeatures.service_delete)},
+      serviceReconfigure: {show: angular.isDefined(productFeatures.service_reconfigure)},
+      serviceRetireNow: {show: angular.isDefined(productFeatures.service_retire_now)},
+      serviceRetire: {show: angular.isDefined(productFeatures.service_retire)},
+      serviceOwnership: {show: angular.isDefined(productFeatures.service_ownership)},
+    };
+
+    return features;
```

Comment:
Still returning on a setter, unless this isn't a setter, in which case it should be renamed.